src/Console/Command/ModuleRegisterCommand.php

```diff
@@ -0,0 +1,46 @@
+<?php
+declare(strict_types=1);
+
+namespace LotGD\Core\Console\Command;
+
+use LotGD\Core\Bootstrap;
```

Comment:
Ah yes, will do. We should probably enforce sorting them too. Ill create an issue for making a linter that enforces this.